Casks/k/koodo-reader.rb

```diff
@@ -1,11 +1,12 @@
 cask "koodo-reader" do
-  arch arm: "-arm64"
+  arch arm: "arm64"
+       intel: "x64"
```

Comment:
```suggestion
  arch arm: "arm64", intel: "x64"
```

Sorry, I read this wrong the last time around.

Reply:
ah I already did the modification on my end sorry